pyproject.toml

```diff
@@ -41,7 +41,7 @@ dependencies = [
     "ruamel.yaml.clib>=0.2.0,<=0.2.8",
     "prompt-toolkit>=3.0.24,<3.0.39",
     "distro>=1.5.0,<1.9.0",
-    "awscrt==0.25.4",
+    "awscrt==0.25.6",
```

Comment:
From that internal chat, should we take this to 0.25.7 now for the improved debugging?

Reply:
Correct, I'm bumping that version and testing with an internal release.